components/system/components/Buttons.js

```diff
@@ -306,3 +306,143 @@ export const ButtonWarning = (props) => {
     />
   );
 };
+
+
+const STYLES_BUTTON_DELETE = css`
```

Comment:
I think we already have a similar button called ButtonWarning. See if you can use that instead of creating a one-off button. Also so that we can keep styles consistent across the UI